webapp/portlet/src/main/webapp/vue-apps/links/components/settings/LinkFormDrawer.vue

```diff
@@ -204,7 +204,8 @@ export default {
           }
         }, 200);
       }
-    },
+      this.$el.closest('#stickyBlockDesktop').style.position = 'static';
```

Comment:
@GouadriaHanen @rdenarie 

Should we add all links portlet inside containers having `stickyBlockDesktop` all time ? What if we add Links portlet outside `stickyBlockDesktop` container ? This container id exists in Activity Stream only if I remember well.

To fix the display of drawer when sticky, please use this workaround instead, which mounts the drawer outside the sticky block:

https://github.com/Meeds-io/gamification/blob/2.5.0/portlets/src/main/webapp/vue-app/usersLeaderboardCommon/components/ProgramSelectorDrawer.vue#L99-L101

```
  mounted() {
    document.querySelector('#vuetify-apps').appendChild(this.$el);
  },
```

Reply:
@mkrout can you check the @boubaker 's proposition ? should we adapt the fix ?

Thanks